src/pages/terminate-test/terminate-test.html

```diff
@@ -1,21 +1,22 @@
-<ion-header>
```

Comment:
This component is at `src/pages`, I think we need a better place to put it, it's not a page after all.

Reply:
No problem. I could place it in ```src/components```, or create a ```src/modal``` directory?

Reply:
I'd say `src/components`. 👍